src/test/fuzz/parse_hd_keypath.cpp

```diff
@@ -18,6 +18,6 @@ FUZZ_TARGET(parse_hd_keypath)
 
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
     const std::vector<uint32_t> random_keypath = ConsumeRandomLengthIntegralVector<uint32_t>(fuzzed_data_provider);
-    (void)FormatHDKeypath(random_keypath);
+    (void)FormatHDKeypath(random_keypath, false);
```

Comment:
Since you changed the default for `apostrophe` in `FormatHDKeypath` to `false` this should be set to `true` here (as the below `WriteHDKeypaht` will already call it with `false`).